app/styles/app.scss

```diff
@@ -0,0 +1,87 @@
+$gutter: 1rem;
+
+body {
+  margin: 0;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+figure {
+  list-style: none;
+}
+
+.app {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+
+}
+
+.app-nav {
+  width: 100%;
+  background: #e2f7fa;
+}
+
+.container {
+  max-width: 65rem;
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+}
+
+.app-main {
+  background: #000;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  overflow: auto;
+
+}
+
+.brewery-list {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.brewery-list__item {
+  width: calc(100% / 3 - 2rem);
+  margin: 0.5rem;
```

Comment:
Make sure your calc is only 2X the margin. Here you have 4X so it's making your grid items too small and not taking up the full row width.